client/settings/payment-methods/index.js

```diff
@@ -81,7 +83,12 @@ const PaymentMethodsPanel = ( { onSaveChanges } ) => {
 			) }
 			<SettingsSection Description={ PaymentMethodsDescription }>
 				<DisplayOrderCustomizationNotice />
-				<GeneralSettingsSection onSaveChanges={ onSaveChanges } />
+				<GeneralSettingsSection
+					onSaveChanges={ onSaveChanges }
+					showLegacyExperienceTransitionNotice={
+						promotionalBannerType !== 'new-checkout-experience'
```

Comment:
Maybe it is worth moving `new-checkout-experience` (and the other types) to constants.

Reply:
Done in [1add408](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3562/commits/1add4083ad5a5d0ccfd9b83687bc95ddd02ba389)